universe/ValueRef.h

```diff
@@ -132,6 +132,16 @@ template<typename T>
     return retval;
 }
 
+template<typename T>
+[[nodiscard]] inline std::vector<std::pair<std::string, std::unique_ptr<T>>> CloneUnique(const std::vector<std::pair<std::string, std::unique_ptr<T>>>& vec) {
+    std::vector<std::pair<std::string, std::unique_ptr<T>>> retval;
+    retval.reserve(vec.size());
+    for (const auto& val : vec) {
+        retval.push_back(std::make_pair(val.first, CloneUnique(val.second)));
```

Comment:
should be able to `emplace_back` without `make_pair`.

Reply:
Fixed.